invenio_theme/templates/semantic-ui/invenio_theme/page.html

```diff
@@ -115,6 +115,19 @@
       {%- endblock body_inner %}
 
       {%- block javascript %}
+
+        {% if config.THEME_MATHJAX_CDN %}
```

Comment:
Why not adding this to the file `javascript.html` instead?

Reply:
I think it is because if we override javascript.html we need to remember to recreate it in the overridden file